docs/core/usage.qmd

```diff
@@ -708,20 +708,20 @@ derivative       | Float64  | -        | -
 The basin table contains:
 
 - results of the storage and level of each basin, which are instantaneous values;
-- results of the vertical fluxes on each basin, which are mean values over the `saveat` intervals. In the time column the start of the period is indicated. This means that for the final time in the table the mean vertical fluxes are not computed, and thus are `missing`.
+- results of the vertical fluxes on each basin, which are mean values over the `saveat` intervals. In the time column the start of the period is indicated.
```

Comment:
Maybe also explicitly indicate here that the final state of the model is absent?